portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/xss/XSS.testcase

```diff
@@ -25,21 +25,25 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LPS-90727."
+	@description = "This is a use case for LPS-90727. Assert no XSS vulnerability when modifying a specific URL from the control panel"
 	@priority = "5"
 	test AssertXSSVulnerabilityInControlPanel {
-		var portalURL = PropsUtil.get("portal.url");
+		task ("Go to Account Settings and save p_p_auth param from the URL") {
+			var portalURL = PropsUtil.get("portal.url");
```

Comment:
Is it localhost? Maybe we could remove this and hardcode the URL in the steps below?